src/Compilers/Core/Portable/Operations/Operation.Enumerable.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections;
+using System;
+using System.Diagnostics;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal abstract partial class Operation : IOperation
+    {
+        internal readonly struct Enumerable : IEnumerable<IOperation>
+        {
+            private readonly Operation? _operation;
```

Comment:
>Operation? [](start = 29, length = 10)

Why nullable?

Reply:
The idea was to ensure that we handle `default` instances. I could remove it since we don't have any cases where it's needed.

---
In reply to: [539583374](https://github.com/dotnet/roslyn/pull/49804#discussion_r539583374) [](ancestors = 539583374)